pkg/oc/cli/cmd/set/env.go

```diff
@@ -124,8 +128,12 @@ func NewCmdEnv(fullName string, f *clientcmd.Factory, in io.Reader, out, errout
 		Long:    envLong,
 		Example: fmt.Sprintf(envExample, fullName),
 		Run: func(cmd *cobra.Command, args []string) {
-			kcmdutil.CheckErr(options.Complete(f, cmd, args))
-			err := options.RunEnv(f)
+			options.Complete(f, cmd, args)
+			err := options.Validate(cmd, args)
```

Comment:
`kcmdutil.CheckErr(options.Validate(cmd, args))` is the preferred way.

Reply:
done